.github/ISSUE_TEMPLATE/metric_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Metric Request
+about: Create a request for metrics/KPIs to be measured
+labels: metric
+---
+
+**Is your metric request related to a problem? Please describe.**
```

Comment:
Is this section needed? Do we typically think of metrics as being associated with a problem other than "Its a problem I don't have access to this metric?" Maybe add a section "associated OKRs" or include that instruction in the Definition section below?  WDYT?

Reply:
yes makes sense, Ill include it in the definition section :+1: